DESCRIPTION

```diff
@@ -22,14 +22,16 @@ Imports:
   broom (>= 0.3.6),
   boot,
   beeswarm,
-  stringr
+  stringr,
+  methods,
+  latex2exp,
+  BayesFactor
```

Comment:
I suspect you made this change to fix checks that I broke. I wonder why it is necessary to import this package while all other packages can be supported without importing. I would very much like to avoid importing BayesFactor due to the multitude of associated dependencies and because it shouldn't be necessary. I'll take a closer look at this myself but could you move this back to the suggests?

Reply:
Done. Yes, this was a fast tweak to get meaningful output from `R CMD CHECK`.

Reply:
I'm confused. `apa_print`-methods for `BFBayesFactor` use functions from the BayesFactor package (`posterior` and `as.BFBayesFactor`, we also have a class import for BFBayesFactor), so I thought importing them would require the BayesFactor package to be listed under `Imports:` in the Description file -- or am I getting something wrong? (Hadley Wickham [wrote](http://r-pkgs.had.co.nz/namespace.html): "The converse is not true. Every package mentioned in NAMESPACE must also be present in the Imports or Depends fields.")

For the sake of having a "clean" chain, I commented out the import and deleted `apa_print.BFBayesFactor.R`.